src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -306,9 +304,24 @@ def _execute(self, rseList):
     def cleanRSE(self, rse):
         """
         The method to implement the actual deletion of files for an RSE.
+        Order of deletion attempts is:
```

Comment:
It would be beneficial to clearly define logic of deletion, here you outline the steps instead of logic. But the underlying code seems very complex with different rules of what and how it should delete. I suggest to provide description of such logic in the doc string.